cloning-instance-with-rest/README.md

```diff
@@ -1,70 +1,105 @@
-Now that you have the script installed run the following command in terminal:
-`appidc -h`  
-This will display the command help as following:  
+### Obtaining a service ID and API-key
 
+1. As an account owner, log into the IBM Cloud console and [create a new service ID](https://console.bluemix.net/docs/iam/serviceid.html#serviceids). Navigate to **Manage > Security > Identity and Access** and then select **Service IDs**. For a detailed walkthrough of creating and managing service IDs, check out [this blog](https://www.ibm.com/blogs/bluemix/2017/10/introducing-ibm-cloud-iam-service-ids-api-keys/).
+
+2. Create an API-key for the service ID that you created previously. You'll need to pass this key to the DevOps person who operates the cloning task.
+
+		>Tip: For simplicity, this example shows how to clone an instance in the same account as the source. To clone to another account, you can create 2 service IDs and API-keys; one in your source account and the other in your target account. 
+
+
+## Assigning access policies
+
+2. Create an access policy in the source instance by assigning a user the service role of `Reader`.
+
+		>Tip: Be sure that you only give a user the rights to complete the tasks that they are assigned. If you give to little, you'll need to adjust it for them to do their job. If you give too much, they may affect more than they intend to. To learn more about App ID actions and roles, check out our doc on [service access management](https://console.bluemix.net/docs/iam/serviceidaccess.html#serviceidpolicy).
+
+3. In the target instance, assign another policy of `writer` for the same person.
+
+
+## Installing the cloning script
+
+Working with the CLI, complete the following steps.
+
+1. Clone the repository.
+	```
+	git clone https://github.com/IBM-Cloud/appid-samples.git
+	```
+2. Change into the script folder.
+	```
+	cd appid-samples/cloning-instance-with-rest
+	```
+3. Install the script.
+	```
+	sudo python setup.py install
+	```
+
+## Running the cloning script
+
+1. Review the configurable parameters.
+2. Run the following command to start the script.
+	```
+	appidc <source_id> <target_id> -apikey <api_key> -region <source_region> -target_region <target_region>
+	```
+
+<table>
+	<tr>
+		<th>Parameter</th>
+		<th>Explanation</th>
+	</tr>
+	<tr>
+		<td><i>source_id</i></td>
+		<td>The tenantID of the source instance of App ID that you want to clone.</td>
+	</tr>
+	<tr>
+		<td><i>target_id</i></td>
+		<td>The tenantID of the target instance of App ID that you want to copy the configuration to.</td>
+	</tr>
+	<tr>
+		<td><i>api_key</i></td>
+		<td>The API key used to create an IAM token with sufficient privileges.</td>
+	</tr>
+	<tr>
+		<td><i>source_region</i></td>
+		<td>The region that the instance of App ID that you want to clone is located in.</td>
+	</tr>
+	<tr>
+		<td><i>target_region</i></td>
+		<td>The region that the instance of App ID that you want to copy the configuration to is located in.</td>
```

Comment:
"The region that the instance of App ID that you want to copy the configuration to is located in. If not provided this will be the same as source region"